modules/sql-engine/src/testFixtures/java/org/apache/ignite/internal/sql/engine/util/SqlTestUtils.java

```diff
@@ -351,4 +355,50 @@ public static RexNode generateLiteralOrValueExpr(ColumnType type, Object value)
                 throw new IllegalArgumentException("Unexpected type: " + type);
         }
     }
+
+    /** Convert {@link ColumnType} to {@link SqlTypeName}. */
+    public static SqlTypeName columnType2SqlTypeName(ColumnType columnType) {
+        switch (columnType) {
+            case NULL:
+                return SqlTypeName.NULL;
+            case BOOLEAN:
+                return SqlTypeName.BOOLEAN;
+            case INT8:
+                return SqlTypeName.TINYINT;
+            case INT16:
+                return SqlTypeName.SMALLINT;
+            case INT32:
+                return SqlTypeName.INTEGER;
+            case INT64:
+                return SqlTypeName.BIGINT;
+            case FLOAT:
+                return SqlTypeName.REAL;
+            case DOUBLE:
+                return SqlTypeName.DOUBLE;
+            case DECIMAL:
+            case NUMBER:
+                return SqlTypeName.DECIMAL;
+            case DATE:
+                return SqlTypeName.DATE;
+            case TIME:
+                return SqlTypeName.TIME;
+            case DATETIME:
+                return SqlTypeName.TIMESTAMP_WITH_LOCAL_TIME_ZONE;
+            case TIMESTAMP:
+                return SqlTypeName.TIMESTAMP;
+            case UUID:
+            case BITMASK:
+                return SqlTypeName.OTHER;
```

Comment:
There is no type that uses SqlTypeName.OTHER. UUID uses ANY, and there is no SqlType for BITMASK.

Reply:
fixed for UUID. For BITMASK it doesn't matter, we don't support the type and it should be get rid under IGNITE-22283